src/test/java/seedu/address/model/tag/MemberTest.java

```diff
@@ -16,16 +16,31 @@ public void constructor_null_throwsNullPointerException() {
 
     @Test
     public void constructor_invalidMemberName_throwsIllegalArgumentException() {
-        String invalidMemberName = "";
+        final String invalidMemberName = "";
         assertThrows(IllegalArgumentException.class, () -> new Member(invalidMemberName));
+
+        final String invalidMemberNameTwo = "    ";
+        assertThrows(IllegalArgumentException.class, () -> new Member(invalidMemberNameTwo));
+
+        final String invalidMemberNameThree = "mary/";
+        assertThrows(IllegalArgumentException.class, () -> new Member(invalidMemberNameThree));
     }
 
     @Test
-    public void isValidMemberName() {
+    public void isValidName_null_throwsNullPointerException() {
         // null tag name
         assertThrows(NullPointerException.class, () -> Member.isValidName(null));
     }
 
+    @Test
+    public void isValidName_invalidMemberName_false() {
+        final String invalidMemberNameTwo = "    ";
+        assertFalse(Member.isValidName(invalidMemberNameTwo));
+
+        final String invalidMemberNameThree = "mary/";
+        assertFalse(Member.isValidName(invalidMemberNameThree));
+    }
+
```

Comment:
Same comments as above.

```suggestion
    @Test
    public void isValidName_invalidBlankMemberName_false() {
        final String invalidMemberNameTwo = "    ";
        assertFalse(Member.isValidName(invalidMemberNameTwo));
    }
    
    @Test
    public void isValidName_invalidSlashMemberName_false() {
        final String invalidMemberNameThree = "mary/";
        assertFalse(Member.isValidName(invalidMemberNameThree));
    }

```